docs/introVideos/basics.md

```diff
@@ -0,0 +1,30 @@
+---
+Order: 2
+Area: introvideos
+TOCTitle: Basics and Setup
+ContentId: baf150cd-6daf-4604-87db-a7c70a6706a7
+PageTitle: Basics and Setup of VS Code
+DateApproved: 8/10/2016
+MetaDescription: Download and learn the basics of Visual Studio Code.
+---
+
+# Basics and Setup of VS Code
+
+In this tutorial we walk you through the basics of Visual Studio Code. 
+
+<iframe src="https://www.youtube.com/embed/fpqy0ZkavWM?rel=0&amp;disablekb=0&amp;modestbranding=1&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+## Outline
+
+* Download and Install VS Code
```

Comment:
Download and install VS Code
Practice **Quick Open** and using the **Command Palette**